src/collective/fingerpointing/browser/logview.pt

```diff
@@ -23,7 +23,9 @@
 
       <div id="content-core">
         <div id="layout-contents">
-          <pre tal:content="view/log_tail"></pre>
+          <pre><span tal:repeat="item view/log_tail" tal:omit-tag=""
```

Comment:
I think this can be enhanced: the repeat statement could be placed inside the `<pre>` tag and you can get rid of that extra `<span>` tag.